Documentation/proposals/0001/README.md

```diff
@@ -0,0 +1,400 @@
+---
+id: 0001
+title: How to write a Proposal
+status: published
+authors: Michael Deffenbaugh <mike@zeroent.net>
+---
+This proposal workflow was taken directly from the [tinkerbell project](https://github.com/tinkerbell/proposals) and full credit goes to them.  Visit the linked repo for more details.
+## Summary
+
+_(**These notes in parenthesis describe the purpose of each section.** The
+summary contains a short summary of the Proposal. Ideally it has to be useful for
+people who do not know what problem the Proposal is attempting to solve. The summary
+gives readers context and helps them evaluate if this is a topic that interests
+or affects them.)_
+
+This is the first Proposal and it covers:
+
+* What Proposals are and why we need them
+* How to write an Proposal
+* When to avoid a Proposal
+* Lifecycle of an Proposal
+
+Rename the [proposal](https://github.com/tinkerbell/proposals) repository to Proposal.
```

Comment:
it can probably be removed?

Reply:
Are you saying just line 23 or also 20 and 21; give this proposal details the proposal lifecycle process, we'd likely want to leave L21.  Obviously pulling out L23

Reply:
@simonpasquier thoughts?

Reply:
I'm guessing just L23.

Reply:
resolved in 65ab2b951cfafd7c8b8c27a427369ebab5910081